tests/src/Traits/ClientTestTrait.php

```diff
@@ -28,7 +26,7 @@ trait ClientTestTrait
     protected array $clientHistory = [];
 
     /**
-     * @param array<string, mixed> $configuration
+     * @param array<mixed> $configuration
```

Comment:
its also numbers so I removed the string in front

Reply:
In this case I was adding `string|int`, as PHPStorm complains that `array<mixed>` is equivalent to `array()`, and after changing this, PHPStan complains that `array()` is not allowed. But this is of tiny importance.